client/components/data/query-atat-eligibility/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { flowRight as compose } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { requestEligibility } from 'state/automated-transfer/actions';
+
+export class QueryAutomatedTransferEligibility extends Component {
+	static propTypes = {
+		request: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
```

Comment:
handled in 26bd9d2 - thanks @gwwar! I'm so helpful for your :eyes: and taking the ⏲ to check on this